test/simple/test-tls-cipher-list.js

```diff
@@ -19,36 +19,21 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-var spawn = require('child_process').spawn;
+var spawn  = require('child_process').spawn;
```

Comment:
Style: the current style used does not align the `=` operator for assignment statements.